viewer/content/faq/03_data_contents.md

```diff
@@ -54,3 +54,14 @@ Defined Daily Dose (DDD) is a unit of measure for medicines consumption that ena
 > Defined Daily Dose (DDD): The assumed average maintenance dose per day for a drug used for its main indication in adults
 
 *Source: [World Health Organization](https://www.who.int/tools/atc-ddd-toolkit/about-ddd)*
+
+### What is indicative cost?
+
+The SCMD includes the indicative cost for each medicine or device within the dataset. The indicative cost is based on the community pharmacy reimbursement prices for generic and list prices for branded medicines. This does not reflect the actual cost paid by hospitals. This, as described in the SCMD release guidance, overestimates the total spend by a factor of two:
```

Comment:
should we make it clear that the factor of 2 does not spread evenly across all meds? 

The risk is someone reads it and divides by 2 to get actual cost of paracetamol